frontend/js/integrations/search.js

```diff
@@ -174,7 +183,9 @@ define([
                     videos.map(function (track) {
                         return {
                             src: track.url,
-                            type: track.mimetype
+                            type: track.mimetype,
+                            framerate: track.video.framerate,
```

Comment:
This also belongs to an extension, which is not part of this merge but seems to have slipped in, though I would suggest to leave it as it is, it may be useful anyway (we needed to hand over some more details for an extension)